pkgs/by-name/gi/gitea-mcp-server/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitea,
+}:
+buildGoModule rec {
+  pname = "gitea-mcp-server";
+  version = "0.2.0";
+
+  src = fetchFromGitea {
+    domain = "gitea.com";
+    owner = "gitea";
+    repo = "gitea-mcp";
+    tag = "v${version}";
+    hash = "sha256-ZUnpE25XIYzSwdEilzXnhqGR676iBQcR2yiT3jhJApc=";
+  };
+
+  vendorHash = "sha256-u9jIjrbDUhnaaeBET+pKQTKhaQLUeQvKOXSBfS0vMJM=";
+
+  subPackages = [ "." ];
+
+  doCheck = false; # no test
+
+  postInstall = ''
+    install -Dm644 README.md LICENSE -t $out/share/gitea-mcp-server
+  '';
+
```

Other MCP Servers have line too, so it prob needs permission for the local networking too. (https://github.com/NixOS/nixpkgs/blob/67d6ef8521f52bf9f0d479b1bceb660faae79639/pkgs/by-name/gi/github-mcp-server/package.nix#L29)
```suggestion
  __darwinAllowLocalNetworking = true;
```